test/api/dependency/api_test.go

```diff
@@ -0,0 +1,70 @@
+package dependency
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/api"
+	"github.com/konveyor/tackle2-hub/test/assert"
+)
+
+func TestDependencyCRUD(t *testing.T) {
+	for _, sample := range Samples {
+		t.Run(fmt.Sprintf("Dependency from %s -> %s", sample.ApplicationFrom.Name, sample.ApplicationTo.Name), func(t *testing.T) {
+
+			assert.Should(t, Application.Create(&sample.ApplicationFrom))
+			assert.Should(t, Application.Create(&sample.ApplicationTo))
+
+			// Create.
+			dependency := api.Dependency{
+				From: api.Ref{
+					ID: sample.ApplicationFrom.ID,
+				},
+				To: api.Ref{
+					ID: sample.ApplicationTo.ID,
+				},
+			}
+			assert.Should(t, Dependency.Create(&dependency))
+
+			// Get.
+			got, err := Dependency.Get(dependency.ID)
+			if err != nil {
+				t.Errorf(err.Error())
+			}
+			if assert.FlatEqual(got, dependency.ID) {
+				t.Errorf("Different response error. Got %v, expected %v", got, dependency.ID)
+			}
+
+			// Delete dependency.
+			assert.Should(t, Dependency.Delete(dependency.ID))
```

Comment:
I missed it meanwhile, but after Delete, there should be a Get testing the Dependency was really deleted, example: https://github.com/konveyor/tackle2-hub/blob/main/test/api/identity/api_test.go#L48-L51